Dockerfile.builder

```
@@ -1,4 +1,4 @@
-FROM golang:1.25.3
+FROM golang:1.25.10
```

RuboCop (1.87.0)

[fatal] no .<digit> floating literal anymore; put 0 before dot
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)